sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 7.0.0b4 (2020-07-06)
 
+**New Features**
+
+* Added support for management of topics, subscriptions, and rules.
```

Comment:
```suggestion
* Added support for management of topics, subscriptions and rules.
```

extra comma :P

Reply:
This is called the oxford comma and is technically correct. :)  I personally find it adds to clarity but won't bikeshed this if we have a principle otherwise.

Reply:
new thing learned! thanks for being my English teacher!